full/Dockerfile

```diff
@@ -9,15 +15,17 @@ RUN apk add rustup
 # Adding cargo binaries to PATH
 ENV PATH=${CARGO_HOME}/bin:${PATH}
 
-# Installing rustup and setting the default toolchain to the 1.67.0 stable channel.
-RUN rustup-init --default-toolchain 1.67.0 -y
+# Installing rustup and setting the default toolchain to the RUST_STABLE_VERSION stable channel.
+RUN rustup-init --default-toolchain $RUST_STABLE_VERSION -y
 
-# Installing nightly channel version 1.67.0 as well.
-RUN rustup install nightly-2022-12-02
+# Installing nightly channel version RUST_NIGHTLY_VERSION as well.
+RUN rustup install $RUST_NIGHTLY_VERSION
 
 # Adding ARMV6M target to the installed toolchains
-RUN rustup target add thumbv6m-none-eabi --toolchain 1.67.0
-RUN rustup target add thumbv6m-none-eabi --toolchain nightly-2022-12-02
+RUN rustup target add thumbv6m-none-eabi --toolchain $RUST_NIGHTLY_VERSION
 
 # Adding rust-src component to nightly channel
-RUN rustup component add rust-src --toolchain nightly-2022-12-02
+RUN rustup component add rust-src --toolchain $RUST_NIGHTLY_VERSION
+
+# Add cargo ledger (needs a version of Rust >= 1.70)
+RUN cargo install --locked --git=https://github.com/LedgerHQ/cargo-ledger.git --rev de68860fb4f0c861f112d0726d3f9f094e1a4c61 cargo-ledger
```

Comment:
Can't we properly version cargo-ledger with a tag at least ?

Reply:
@yhql What do you think? We could tag v1.1.0 for example?

Reply:
Done